### PR TITLE
Registration

### PR DESCRIPTION
**Pull request by annsudo**

### CODE REVIEW

**Comment by khesoem** [comment]
Hi,
I think the topic is relevant. A little more detail is needed.

File: contributions/presentation/week2/annnik-carllei/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+## Members
+Anna Nikolskaya (annnik@kth.se)
+Github username: annsudo
+
+Carl Leijonberg (carllei@kth.se)
+Github username: carllei
+
+## Topic "DevSecOps throughout the DevOps lifecycle"
+During the presentation we will talk about the importance of DevSecOps throughout different stages of DevOps lifecycle.  We will discuss what are the current best practices in DevSecOps.
```
Comment (khesoem) [inline contributions/presentation/week2/annnik-carllei/README.md]: - a short statement (1-3 sentences) about the relevance and timeliness for testing and devops
- bullet outline of the presentation

Don't forget to add these two.

**Comment by annsudo** [comment]
Hi,
Thank you for fast reply. Can we add details closer to the presentation, or do we need to add them asap in terms of geting the spot for week2?

**Review by khesoem — CHANGES_REQUESTED** [review]
A little more information should be provided in your proposal.

Also, please try remove the `.DS_Store` file

**Comment by khesoem** [comment]
> Hi,
> Thank you for fast reply. Can we add details closer to the presentation, or do we need to add them asap in terms of geting the spot for week2?

To get your PR merged, you have to add the information. So, I guess you should do it ASAP to get the spot for week2.

**Comment by annsudo** [comment]
Done!

**Comment by khesoem** [comment]
Hi @annsudo 
I just found this PR from previous years:
[https://github.com/KTH/devops-course/pull/435](https://github.com/KTH/devops-course/pull/435)

I think your proposal is too close to that one. You can make your proposal more concrete to make it different from that one.

**Comment by annsudo** [comment]
Hi, I looked through that topic and our topics are about different things. Our group is not talking about Kubernetes at all. I wonder what should we change?

**Comment by khesoem** [comment]
You are right, their topic is more specific. But they also talk about DevSecOps and the fact that they discuss DevSecOps in context of Kubernetes makes their topic more concrete and therefore more interesting.

I think you can make your work different from theirs by focusing more the third section of your agenda "Secure CI/ CD Pipelines (+ some tools)" and discuss a specific tool their.

WDYT?

**Comment by annsudo** [comment]
Thank you for the recommendation, have reformulated it now

**Comment by annsudo** [comment]
Hi again, we made new changes where we narrow topic more. 
Yet I pulled it as new request:Presentation week2 #1005

**Pull request closed without merging**